### ISSUE TITLE
Need of SimSPFieldUrlValue

### ISSUE THREAD

**Issue by valmaev** [wulfland/SPEmulators#4]
Why you're faking SPFieldUrlValue? We can create instance of it without any shims. I ask about this case because it prevents AutoFixture from generating SPFieldUrlValue instances.

**Comment by wulfland** [comment]
I'm sorry - I totally missed this issue. Could you solve your problem or does it still exist?

**Comment by valmaev** [comment]
It's still exists. I can create pull request that would delete SimSPFieldUrlValue because in my opinion it's useless. As I said, we can create SPFieldUrlValue instance without shim magic.

**Comment by wulfland** [comment]
Makes sense. If you create the pull request I merge the changes and create a new package.
